dotcom-rendering/src/components/FixedMediumSlowXIIMPU.tsx

```diff
@@ -106,15 +106,15 @@ const ThreeColumnSliceWithAdSlot = ({
 				 * |_______________________|
 				 */}
 				<UL direction="row" wrapCards={true}>
-					{trails.map((trail, trailIndex, { length }) => (
+					{trails.map((trail, trailadIndex, { length }) => (
```

Comment:
What’s trailad 😉 – even urbandictionary.com/trailad is of no help

Reply:
oops! 🙊